clang/test/Preprocessor/sycl-macro.cpp

```diff
@@ -30,3 +31,5 @@
 
 // CHECK-NO-SYCL_FIT_IN_INT-NOT:#define __SYCL_ID_QUERIES_FIT_IN_INT__ 1
 // CHECK-SYCL-ID:#define __SYCL_ID_QUERIES_FIT_IN_INT__ 1
+
+// CHECK-CUDA:#define __CUDA_ARCH__ 800
```

Comment:
Can you add newline here and in sycl-bfloat16-lib.cpp

Reply:
OK, done.